sky/packages/sky/lib/src/widgets/snack_bar.dart

```diff
@@ -38,7 +38,7 @@ class SnackBarAction extends Component {
   }
 }
 
-class SnackBar extends Component {
+class SnackBar extends StatefulComponent {
```

Comment:
@mpcomplete How do you feel about a base class like AnimatedComponent that you'd inherit from, which manages an AnimationPerformance? SnackBar would then inherit from that (and just pass the duration in the superclass constructor).